x-pack/docs/en/rest-api/security/create-cross-cluster-api-key.asciidoc

```diff
@@ -0,0 +1,9 @@
+[role="xpack"]
+[[security-api-create-cross-cluster-api-key]]
+=== Create Cross-Cluster API key API
+
+++++
+<titleabbrev>Create Cross-Cluster API key</titleabbrev>
+++++
+
+TODO: Placeholder
```

Comment:
Need placeholders for the doc pages. Otherwise the doc build will fail because the REST spec files reference non-existent URLs.